Packs/ApiModules/Scripts/MicrosoftAzureStorageApiModule/MicrosoftAzureStorageApiModule.py

```diff
@@ -0,0 +1,119 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+
+class MicrosoftStorageClient(BaseClient):
+    """
+    Microsoft Azure Storage API Client
+    """
+
+    def __init__(self, server_url, verify, proxy, account_sas_token, storage_account_name, api_version):
+        super().__init__(base_url=server_url, verify=verify, proxy=proxy)
+        self._account_sas_token = account_sas_token
+        self._storage_account_name = storage_account_name
+        self._api_version = api_version
+        self._base_url = server_url
+
+    def http_request(
+            self, *args, method, url_suffix="", params=None, resp_type='response', headers=None,
```

Comment:
MS requests contain a variety of types of return values, I preferred to return the response as a default value because this return value can be found in a wide variety of requests compared to JSON.